api/models/containerimageinfo.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+// GetRepoTag returns the first repo tag if it exists. Otherwise, returns false.
+func (c *ContainerImageInfo) GetRepoTag() (string, bool) {
+	var tag string
+	var ok bool
+
+	if len(*c.RepoTags) > 0 {
```

Comment:
need to check for c == nil && c.c.RepoTags == nil ?

Reply:
@fishkerez We can check if `c` is `nil` to make sure the receiver is not `nil`, however `len()` returns `0` if `c.RepoTags` is `nil` so the 2nd check is not needed as `len` does that implicitly.

Reply:
yeah, but we are dereferencing c.RepoTags (by doing *c.RepoTags), so if c.RepoTags are nil, we will crush

Reply:
Testing the receiver for `nil` is not something we do in other parts of the code base as the assumption is that the caller won't call any method on `nil` pointer. Or at least it makes sure that safeguards are in place to avoid this issue. Personally I do not mind to test the receiver as well, but that we need to make sure that we follow the same principle in other parts of the code base as well.

The problem is that what should be the return value if the the receiver is null? An error or `0` value?

Reply:
> yeah, but we are dereferencing c.RepoTags (by doing *c.RepoTags), so if c.RepoTags are nil, we will crush

That is valid point. @paralta Could you please fix this?